src/godot_fmod.cpp

```diff
@@ -14,147 +16,147 @@ Fmod::~Fmod() {
-    register_method("load_file_as_sound", &Fmod::load_file_as_sound);
-    register_method("load_file_as_music", &Fmod::load_file_as_music);
-    register_method("unload_file", &Fmod::unload_file);
-    register_method("create_sound_instance", &Fmod::create_sound_instance);
-    register_method("check_sound_instance", &Fmod::check_sound_instance);
-    register_method("release_sound", &Fmod::release_sound);
-    register_method("play_sound", &Fmod::play_sound);
-    register_method("stop_sound", &Fmod::stop_sound);
-    register_method("set_sound_paused", &Fmod::set_sound_paused);
-    register_method("is_sound_playing", &Fmod::is_sound_playing);
-    register_method("set_sound_volume", &Fmod::set_sound_volume);
-    register_method("get_sound_volume", &Fmod::get_sound_volume);
-    register_method("set_sound_pitch", &Fmod::set_sound_pitch);
-    register_method("get_sound_pitch", &Fmod::get_sound_pitch);
-    register_method("set_callback", &Fmod::set_callback);
-    register_method("set_sound_3D_settings", &Fmod::set_sound_3d_settings);
-    register_method("wait_for_all_loads", &Fmod::wait_for_all_loads);
-    register_method("get_available_drivers", &Fmod::get_available_drivers);
-    register_method("get_driver", &Fmod::get_driver);
-    register_method("set_driver", &Fmod::set_driver);
-    register_method("get_performance_data", &Fmod::get_performance_data);
-    register_method("set_global_parameter_by_name", &Fmod::set_global_parameter_by_name);
-    register_method("get_global_parameter_by_name", &Fmod::get_global_parameter_by_name);
-    register_method("set_global_parameter_by_id", &Fmod::set_global_parameter_by_id);
-    register_method("get_global_parameter_by_id", &Fmod::get_global_parameter_by_id);
-    register_method("get_global_parameter_desc_by_name", &Fmod::get_global_parameter_desc_by_name);
-    register_method("get_global_parameter_desc_by_id", &Fmod::get_global_parameter_desc_by_id);
-    register_method("get_global_parameter_desc_count", &Fmod::get_global_parameter_desc_count);
-    register_method("get_global_parameter_desc_list", &Fmod::get_global_parameter_desc_list);
-    register_method("set_dsp_buffer_size", &Fmod::set_system_dsp_buffer_size);
-    register_method("get_dsp_buffer_size", &Fmod::get_system_dsp_buffer_size);
-    register_method("get_dsp_buffer_length", &Fmod::get_system_dsp_buffer_length);
-    register_method("get_dsp_num_buffers", &Fmod::get_system_dsp_num_buffers);
-    register_method("_process", &Fmod::_process);
-
-    register_signal<Fmod>("timeline_beat", "params", GODOT_VARIANT_TYPE_DICTIONARY);
-    register_signal<Fmod>("timeline_marker", "params", GODOT_VARIANT_TYPE_DICTIONARY);
-    register_signal<Fmod>("sound_played", "params", GODOT_VARIANT_TYPE_DICTIONARY);
-    register_signal<Fmod>("sound_stopped", "params", GODOT_VARIANT_TYPE_DICTIONARY);
+void Fmod::_bind_methods() {
+    ClassDB::bind_method(D_METHOD("init"), &Fmod::init);

```

Comment:
We might want to use the `ClassDB` namespace here for more readable code, so it would become:
```cpp
bind_method(...);
```
everywhere

Reply:
ClassDB is not a namespace!